packages/x-tailwindcss/src/x-tailwind-plugin/plugin.ts

```diff
@@ -33,7 +33,11 @@ export default plugin.withOptions(
   },
   function (options) {
     return {
-      theme: deepMerge({}, theme, options?.theme)
+      theme: deepMerge(
+        { colors: { current: 'currentColor', transparent: 'transparent' } },
+        theme,
+        options?.theme
+      )
     } as Config;
   }
```

Comment:
Sooner or later I think we should document this file, I'm afraid that I don't completely understand everything going on here.

Reply:
created subtask for that: https://searchbroker.atlassian.net/browse/EX-7129